build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.19-openshift-4.13 as build
+FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.14 as build
```

Comment:
I just heard from @soltysh that there are build issues with the rhel-9 images and for consistency it is best to stick to rhel-8 images. We can switch to rhel-9 along with rest of the org.

Reply:
The change here is to golang 1.20.  We've already been using rhel-9.  Same for 4.13.

Reply:
Yes I understand the change is for golang 1.20. Point I am making is that we should be using` registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.20-openshift-4.14`